german/Alkohol/Schatzkarte/Schatzkarte.md

```diff
@@ -0,0 +1 @@
+Sie erinnern sich, dass Sie sich am Abend zuvor mit Ihren Freunden auf die Suche nach einem versteckten Schatz begeben haben.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.